receiver/receivercreator/config_expansion.go

```diff
@@ -0,0 +1,116 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package receivercreator
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/antonmedv/expr"
+)
+
+func expandConfigValue(env endpointEnv, configValue string) (interface{}, error) {
```

Comment:
Please add a comment to explain what does this expand and how.

Reply:
If I understand the code correctly this evaluates expressions inside backticks and replaces them with the result of evaluation. The name `expandConfigValue` does not quite convey this. Perhaps `evalBackticksInConfigValue`?